### DISCUSSION TITLE
queryRaw

### DISCUSSION THREAD

**Discussion by Ben212** [prisma/prisma#17541] [Q&A]
is it possible to use regexp_matches in queryRaw ? I try this
await prisma.$queryRaw
      `SELECT regexp_matches(name, '^pick\W?up$')
                 FROM "MyTable"`;

doesn't work but work in postgresql console

**Comment by nurul3101** [comment]
Hello @Ben212 👋 

Can you try adding the schema name in front of the table name and then try?
Example: `SELECT regexp_matches(name, '^pick\W?up$') FROM "MySchema"."MyTable"`

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
